packages/cubejs-client-core/src/tests/ResultSet.test.js

```diff
@@ -1224,4 +1224,101 @@ describe('ResultSet', () => {
       );
     });
   });
+
+  describe('tableColumns',()=> {
```

Comment:
Looks good! One small thing, could you please move this test to `table.test.js`?

Reply:
No problem, i just saw the `ResultSet.test.js` and added it here, did not realize that there are more specific test files too.

Just one question: should i add a new sub-describe or can i just add the test in the top level one?

Reply:
@dodgex if it's just a single test there's no need to wrap it in another `describe`

Reply:
Done! :)